src/map/handler/Map.ScrollWheelZoom.js

```diff
@@ -25,6 +25,7 @@ L.Map.ScrollWheelZoom = L.Handler.extend({
 	},
 
 	_onWheelScroll: function (e) {
+		if (this._map.options.scrollWheelZoom === 'shiftkey' && !e.shiftKey) { return false; }
```

Comment:
I must ask why the `return false`. If my memory serves right, the return value of an event handler was used to stop event propagation... but that was many, many years ago. The current documentation says the return value of event handlers is ignored.

Reply:
Yes you are right, return values are ignored. For DomEvent and also for the Leaflet-Events.